web/templates/settings/realm_domains_modal.hbs

```diff
@@ -12,7 +12,7 @@
             <td>
                 <label class="checkbox">
                     <input type="checkbox" class="new-realm-domain-allow-subdomains" />
-                    <span></span>
+                    <span class="rendered-checkbox"></span>
```

Comment:
Probably a follow-up we should be planning is to make most of these use the `settings_checkbox` template as a partial.